### ISSUE TITLE
Setup certain PHP configs to be editable with environment variables

### ISSUE THREAD

**Issue by jonpugh** [terra-ops/docker-drupal#1]
I need to up the PHP memory limit.  I don't want to hard code it into the container.

Let's figure out how to use an environment variable to set it.

**Comment by jlyon** [comment]
+1 for this

From http://stackoverflow.com/questions/858431/is-it-possible-to-use-environment-variables-in-php-ini:
According to the php manual you can access any env var in the php.ini via ${VARNAME}. –  Marcus Pope Mar 17 '14 at 19:09

We'd need to make sure to set the default `$PHP_MEMORY_LIMIT`. Should this be done in terra-app?

**Comment by jonpugh** [comment]
Yep, environment vars are set here: https://github.com/terra-ops/terra-app/blob/master/src/terra/Factory/EnvironmentFactory.php#L305

To override for a specific app, we would use the `docker-compose: overrides: app: environment:` item in .terra.yml...

```
docker_compose:
  overrides:
    app:
      environment:
        PHP_MEMORY_LIMIT: 256MB
```

I'll need to test that the overrides behavior merges based on the environment name.

**Comment by jlyon** [comment]
After rebuilding docker-drupal with 

```
memory_limit=${PHP_MEMORY_LIMIT}
```

I get a fatal PHP error. It seems like `${PHP_MEMORY_LIMIT}` isn't getting set properly.

```
Allowed memory size of 262144 bytes exhausted (tried to allocate 4480 bytes) in /usr/share/nginx/html/includes/bootstrap.inc on line 201"
```

**Comment by jonpugh** [comment]
Did you add to the Docker file?

ENV PHP_MEMORY_LIMIT 128m

That will set the default environment variable.
